ISLE/res/resource.h

```diff
@@ -3,3 +3,8 @@
 #define ISLE_BUSY 104
 
 #define APP_ICON 105
+
+#ifndef _RES_ISLE_H
+#define _RES_ISLE_H
+#include "isle.h"
+#endif
```

Comment:
This feels awkward.
I propose to name the generated `isle.h` `isle_bmp.h`.
Simply include `isle_bmp.h` in `isleapp.cpp`, and leave `resource.h` alone.